.tool-versions

```diff
@@ -1,2 +1,2 @@
-elixir 1.17.1-otp-27
+elixir 1.18.3-otp-27
```

Comment:
There's a dialyzer issue in version 1.17, the version was already updated in CI: https://github.com/turnhub/flow_runner/pull/258/commits/9752d7db571618b643b126df9fc2613320fdfdf4